spec/takeaway_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'takeaway'
+
+#let (:my_order) {Takeaway.new}
+
+describe Takeaway do
+  it 'creating a new instance of takeaway' do
+    expect(Takeaway.new).to be_a Takeaway
+  end
+#see a list of dishes with prices - see = puts, list = an array
+  describe '#menu' do
+   it { is_expected.to respond_to(:menu).with(1).argument }
+
+   it 'dishes hash exists to be seen' do
+      dishes = {
+        "Chicken Korma" => 4.50,
+        "Veggie Burrito" => 3.00,
+        "Fries" => 3.00,
+        "Beef Burger" => 7.00,
+        "Steak" => 7.00,
+        "Kinder Bueno Waffle" => 5.00,
+        "Vanilla Ice Cream" => 5.00
+      }
+      expect(Takeaway.new.menu(dishes)).to include("Chicken Korma" => 4.50)
```

Comment:
Nice, it's great that you have got this test to work! I think you actually don't need to call the `menu` method, as you are defining dishes as a variable in Takeaway. 

so this could be simplified to:

```
takeaway = Takeaway.new
expect(takeaway.dishes).to include("Chicken Korma" => 4.50)
```